src/phpDocumentor/Application/Commands/ContainerLocator.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Application\Commands;
+
+use Interop\Container\ContainerInterface;
+use League\Tactician\Handler\Locator\HandlerLocator;
+use Webmozart\Assert\Assert;
+
+/**
+ * Command Handler Locator for Tactician that can locate handlers based on a provided mapping or by suffixing the FQCN
```

Comment:
120 characters is allowed according to PSR-2, but perhaps we should limit DocBlocks to 80 for readability?